Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM registry.opensource.zalan.do/stups/ubuntu:UPSTREAM
+FROM registry.opensource.zalan.do/stups/ubuntu:latest
```

Comment:
can you change this back to "UPSTREAM" --- this is our special marker which is replaced with the upstream image tag by the CD pipeline (Go CD)

"latest" would also work but we want to have one pipeline using the same artifact